README.md

```diff
@@ -31,6 +31,26 @@ Here is a brief list of the available widgets:
 * **edit_box**: an editable text box (multi-line edit_boxes are not yet fully supported).
 * **scroll_frame**: a frame that has scrollable content.
 
+# Getting started 
+```
+git clone https://github.com/unnamed-mmo/lxgui
+sudo dnf install -y freetype-devel SFML-devel lua-bit32 mesa-libGLU-devel lua-devel
+cd lxgui
+git submodule update --init
+```                                                                       
+## install GLEW                                                                                                    
```

Comment:
This section should be removed for a quick start guide, as GLEW isn't required when using the pure SFML backend.

Reply:
I've broken it out into a basic setup, and an openGL setup! Does it look straightforward to you, with having openGL after the basic setup? I want it to be clear that people need to do one, then the other.

Reply:
Yep, looks good. Thanks.